.travis.yml

```diff
@@ -7,15 +7,21 @@ env:
     - TOXENV=py33
     - TOXENV=py34
     - TOXENV=py35
+    - TOXENV=py36
     - TOXENV=pypy
     - TOXENV=cov
     - TOXENV=sith
 matrix:
-    allow_failures:
-        - env: TOXENV=cov
-        - env: TOXENV=sith
-        - env: TOXENV=pypy
-python: 3.5
```

Comment:
Because "python: 3.5" && TOXENV=py36 fails with InterpreterNotFound: https://travis-ci.org/davidhalter/jedi/jobs/187838411.

Likewise, "python: 3.6" && TOXENV=py35 fails with the same error: https://travis-ci.org/davidhalter/jedi/jobs/187838832.